### DISCUSSION TITLE
Pytorch v1.4 整体进度（校对）

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/pytorch-doc-zh#619] [Announcements]
> 认领须知

```
提交的时候不要改动文件名称，
即使它跟章节标题不一样也不要改，
因为文件名和原文的链接是对应的！！！
```

【校验的贡献者 - 没有任何限制，你主动我们就有故事】
校验的事情人人都可以做，你一边看文档学习，发现错误就可以提交PR
我们在GitHub上面只需要点击合并一下，网站就会自动编译和生成好，完全自动化

* 留言格式: 昵称 + QQ + 章节
* 需要取消认领的也在此留言。（2个月不更新，默认放弃任务）
* 公式问题如何解决: <https://github.com/apachecn/pytorch-doc-zh/issues/360>
* 英文教程：<https://pytorch.org/tutorials/>
* 英文文档：<https://pytorch.org/docs/1.4.0/>
* 中文教程和文档参考v1.0版本：<https://pytorch.apachecn.org/docs/1.0/>
* 模版文章: <https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/chatbot_tutorial.md>
* 知乎中文专栏： <https://zhuanlan.zhihu.com/c_1141766027527385088>
* 图片存放地址： docs/1.4/img   目录下，记得文章也改为相对路径

| 章节   | 校验者 | 进度 |
| --- | --- | --- |
| 中文教程 | - | - |
| 入门 | - | - |
| [使用 PyTorch 进行深度学习：60 分钟的闪电战](http://pytorch.apachecn.org/docs/1.4/4.html) | @Shun-Qiang | 100% |
| [什么是PyTorch？](http://pytorch.apachecn.org/docs/1.4/blitz/tensor_tutorial.html) | @SAGIRI-kawaii |  |
| [Autograd：自动求导](http://pytorch.apachecn.org/docs/1.4/blitz/autograd_tutorial.html) | @GZYZG | 100% |
| [神经网络](http://pytorch.apachecn.org/docs/1.4/blitz/neural_networks_tutorial.html) | @GG-yuki | 100% |
| [训练分类器](http://pytorch.apachecn.org/docs/1.4/blitz/cifar10_tutorial.html) | @yearing1017  | 100% |
| [可选：数据并行](http://pytorch.apachecn.org/docs/1.4/blitz/data_parallel_tutorial.html) | @yearing1017  | 100% |
| [编写自定义数据集，数据加载器和转换](http://pytorch.apachecn.org/docs/1.4/5.html) | @Shun-Qiang | 100% |
| [使用 TensorBoard 可视化模型，数据和训练](http://pytorch.apachecn.org/docs/1.4/6.html) | @yearing1017  | 100% |
| 图片 | - | - |
| [TorchVision 对象检测微调教程](http://pytorch.apachecn.org/docs/1.4/8.html) | @yearing1017 | 100% |
| [转移学习的计算机视觉教程](http://pytorch.apachecn.org/docs/1.4/9.html) | | |
| [空间变压器网络教程](http://pytorch.apachecn.org/docs/1.4/10.html) | @GZYZG | |
| [使用 PyTorch 进行神经传递](http://pytorch.apachecn.org/docs/1.4/11.html) | | |
| [对抗示例生成](http://pytorch.apachecn.org/docs/1.4/12.html) | | |
| [DCGAN 教程](http://pytorch.apachecn.org/docs/1.4/13.html) | | |
| 音频 | - | - |
| [torchaudio 教程](http://pytorch.apachecn.org/docs/1.4/15.html) | | |
| 文本 | - | - |
| [NLP From Scratch: 使用char-RNN对姓氏进行分类](http://pytorch.apachecn.org/docs/1.4/17.html) | @haiyang| |
| [NLP From Scratch: 生成名称与字符级RNN](http://pytorch.apachecn.org/docs/1.4/18.html) | @haiyang| |
| [NLP From Scratch: 基于注意力机制的 seq2seq 神经网络翻译](http://pytorch.apachecn.org/docs/1.4/19.html) | @haiyang| |
| [使用 TorchText 进行文本分类](http://pytorch.apachecn.org/docs/1.4/20.html) | @haiyang| |
| [使用 TorchText 进行语言翻译](http://pytorch.apachecn.org/docs/1.4/21.html) | @haiyang| |
| [使用 nn.Transformer 和 TorchText 进行序列到序列建模](http://pytorch.apachecn.org/docs/1.4/22.html) | @zanshuxun | 100% |
| 命名为 Tensor(实验性） | @haiyang| - |
| [(实验性）PyTorch 中的命名张量简介](http://pytorch.apachecn.org/docs/1.4/24.html) | @hyl11 | |
| 强化学习 | - | - |
| [强化学习(DQN）教程](http://pytorch.apachecn.org/docs/1.4/26.html) | @Dynmi | 100% |
| 在生产中部署 PyTorch 模型 | - | - |
| [通过带有 Flask 的 REST API 在 Python 中部署 PyTorch](http://pytorch.apachecn.org/docs/1.4/28.html) | | |
| [TorchScript 简介](http://pytorch.apachecn.org/docs/1.4/29.html) | | |
| [在 C ++中加载 TorchScript 模型](http://pytorch.apachecn.org/docs/1.4/30.html) | | |
| [(可选）将模型从 PyTorch 导出到 ONNX 并使用 ONNX Runtime 运行](http://pytorch.apachecn.org/docs/1.4/31.html) | | |
| 并行和分布式训练 | - | - |
| [单机模型并行最佳实践](http://pytorch.apachecn.org/docs/1.4/33.html) | @Shun-Qiang | 100% |
| [分布式数据并行入门](http://pytorch.apachecn.org/docs/1.4/34.html) | @ethan4335 | |
| [用 PyTorch 编写分布式应用程序](http://pytorch.apachecn.org/docs/1.4/35.html) | | |
| [分布式 RPC 框架入门](http://pytorch.apachecn.org/docs/1.4/36.html) | | |
| [(高级）带有 Amazon AWS 的 PyTorch 1.0 分布式训练师](http://pytorch.apachecn.org/docs/1.4/37.html) | | |
| 扩展 PyTorch | - | - |
| [使用自定义 C ++运算符扩展 TorchScript](http://pytorch.apachecn.org/docs/1.4/39.html) | | |
| [使用自定义 C ++类扩展 TorchScript](http://pytorch.apachecn.org/docs/1.4/40.html) | | |
| [使用 numpy 和 scipy 创建扩展](http://pytorch.apachecn.org/docs/1.4/41.html) | | |
| [自定义 C ++和 CUDA 扩展](http://pytorch.apachecn.org/docs/1.4/42.html) | | |
| 模型优化 | - | - |
| [LSTM Word 语言模型上的(实验）动态量化](http://pytorch.apachecn.org/docs/1.4/44.html) | | |
| [(实验性）在 PyTorch 中使用 Eager 模式进行静态量化](http://pytorch.apachecn.org/docs/1.4/45.html) | | |
| [(实验性）计算机视觉教程的量化转移学习](http://pytorch.apachecn.org/docs/1.4/46.html) | | |
| [(实验）BERT 上的动态量化](http://pytorch.apachecn.org/docs/1.4/47.html) | | |
| [修剪教程](http://pytorch.apachecn.org/docs/1.4/48.html) | | |
| PyTorch 用其他语言 | - | - |
| [使用 PyTorch C ++前端](http://pytorch.apachecn.org/docs/1.4/50.html) | @GG-yuki  | 100% |
| PyTorch 基础知识 | - | - |
| [通过示例学习 PyTorch](http://pytorch.apachecn.org/docs/1.4/52.html) | | |
| [torch.nn 到底是什么？](http://pytorch.apachecn.org/docs/1.4/53.html) | @Shun-Qiang | 100% |
| 文件 | - | - |
| 笔记 | - | - |
| [自动毕业力学](http://pytorch.apachecn.org/docs/1.4/56.html) | | |
| [广播语义](http://pytorch.apachecn.org/docs/1.4/57.html) | | |
| [CPU 线程和 TorchScript 推断](http://pytorch.apachecn.org/docs/1.4/58.html) | | |
| [CUDA 语义](http://pytorch.apachecn.org/docs/1.4/59.html) | | |
| [分布式 Autograd 设计](http://pytorch.apachecn.org/docs/1.4/60.html) | | |
| [扩展 PyTorch](http://pytorch.apachecn.org/docs/1.4/61.html) | | |
| [经常问的问题](http://pytorch.apachecn.org/docs/1.4/62.html) | | |
| [大规模部署的功能](http://pytorch.apachecn.org/docs/1.4/63.html) | | |
| [并行处理最佳实践](http://pytorch.apachecn.org/docs/1.4/64.html) | | |
| [重现性](http://pytorch.apachecn.org/docs/1.4/65.html) | | |
| [远程参考协议](http://pytorch.apachecn.org/docs/1.4/66.html) | | |
| [序列化语义](http://pytorch.apachecn.org/docs/1.4/67.html) | | |
| [Windows 常见问题](http://pytorch.apachecn.org/docs/1.4/68.html) | | |
| [XLA 设备上的 PyTorch](http://pytorch.apachecn.org/docs/1.4/69.html) | | |
| 语言绑定 | - | - |
| [PyTorch C ++ API](http://pytorch.apachecn.org/docs/1.4/71.html) | | |
| [PyTorch Java API](http://pytorch.apachecn.org/docs/1.4/72.html) | | |
| Python API | - | - |
| [torch](http://pytorch.apachecn.org/docs/1.4/74.html) | | |
| [torch.nn](http://pytorch.apachecn.org/docs/1.4/75.html) | | |
| [torch功能](http://pytorch.apachecn.org/docs/1.4/76.html) | | |
| [torch.Tensor](http://pytorch.apachecn.org/docs/1.4/77.html) | @Shun-Qiang | 100% |
| [张量属性](http://pytorch.apachecn.org/docs/1.4/78.html) | | |
| [自动差分包-Torch.Autograd](http://pytorch.apachecn.org/docs/1.4/79.html) | | |
| [torch.cuda](http://pytorch.apachecn.org/docs/1.4/80.html) | | |
| [分布式通讯包-Torch.Distributed](http://pytorch.apachecn.org/docs/1.4/81.html) | | |
| [概率分布-torch分布](http://pytorch.apachecn.org/docs/1.4/82.html) | | |
| [torch.hub](http://pytorch.apachecn.org/docs/1.4/83.html) | | |
| [torch脚本](http://pytorch.apachecn.org/docs/1.4/84.html) | | |
| [torch.nn.init](http://pytorch.apachecn.org/docs/1.4/85.html) | | |
| [torch.onnx](http://pytorch.apachecn.org/docs/1.4/86.html) | | |
| [torch.optim](http://pytorch.apachecn.org/docs/1.4/87.html) | | |
| [量化](http://pytorch.apachecn.org/docs/1.4/88.html) | | |
| [分布式 RPC 框架](http://pytorch.apachecn.org/docs/1.4/89.html) | | |
| [torch随机](http://pytorch.apachecn.org/docs/1.4/90.html) | | |
| [torch稀疏](http://pytorch.apachecn.org/docs/1.4/91.html) | | |
| [torch存储](http://pytorch.apachecn.org/docs/1.4/92.html) | | |
| [torch.utils.bottleneck](http://pytorch.apachecn.org/docs/1.4/93.html) | | |
| [torch.utils.checkpoint](http://pytorch.apachecn.org/docs/1.4/94.html) | | |
| [torch.utils.cpp_extension](http://pytorch.apachecn.org/docs/1.4/95.html) | | |
| [torch.utils.data](http://pytorch.apachecn.org/docs/1.4/96.html) | | |
| [torch.utils.dlpack](http://pytorch.apachecn.org/docs/1.4/97.html) | | |
| [torch.utils.model_zoo](http://pytorch.apachecn.org/docs/1.4/98.html) | | |
| [torch.utils.tensorboard](http://pytorch.apachecn.org/docs/1.4/99.html) | | |
| [类型信息](http://pytorch.apachecn.org/docs/1.4/100.html) | | |
| [命名张量](http://pytorch.apachecn.org/docs/1.4/101.html) | | |
| [命名为 Tensors 操作员范围](http://pytorch.apachecn.org/docs/1.4/102.html) | | |
| [糟糕！](http://pytorch.apachecn.org/docs/1.4/103.html) | | |
| torchvision参考 | - | - |
| [torchvision](http://pytorch.apachecn.org/docs/1.4/105.html) | @zhangqizky | |
| 音频参考 | - | - |
| [torchaudio](http://pytorch.apachecn.org/docs/1.4/107.html) | | |
| torchtext参考 | - | - |
| [torchtext](http://pytorch.apachecn.org/docs/1.4/109.html) | | |
| 社区 | - | - |
| [PyTorch 贡献指南](http://pytorch.apachecn.org/docs/1.4/111.html) | | |
| [PyTorch 治理](http://pytorch.apachecn.org/docs/1.4/112.html) | | |
| [PyTorch 治理 感兴趣的人](http://pytorch.apachecn.org/docs/1.4/113.html) | | |

**Comment by Dynmi** [comment]
Dynmi 3391738016 强化学习(DQN）教程

**Comment by Shun-Qiang** [comment]
小枫 + 396694334 + torch.nn 到底是什么？

**Comment by Shun-Qiang** [comment]
小枫 + 396694334 + 单机模型并行最佳实践

**Comment by GG-yuki** [comment]
神经网络，1390867192

**Comment by zhangqizky** [comment]
真想不出昵称+2445657635+torchvision

**Comment by yearing1017** [comment]
yearing1017 + 1205231870 + 训练分类器

**Comment by yearing1017** [comment]
yearing1017 + 1205231870 + 可选：数据并行（已完成100%，请更新）

**Comment by yearing1017** [comment]
yearing1017 + 1205231870 + 使用 TensorBoard 可视化模型，数据和训练

**Comment by ethan4335** [comment]
ethan4335 + 309335077 + 分布式数据并行入门

**Comment by yearing1017** [comment]
yearing1017 + 1205231870 + TorchVision 对象检测微调教程

**Comment by SAGIRI-kawaii** [comment]
SAGIRI + 1900384123 + 什么是PyTorch？

**Comment by lihy11** [comment]
haiyang + 1130248527 + NLP From Scratch: 使用char-RNN对姓氏进行分类（已完成100％）

**Comment by GZYZG** [comment]
GZYZG+ 2227423992 + Autograd：自动求导

**Comment by lihy11** [comment]
haiyang + 1130248527 + 文本部分剩余章节

**Comment by GZYZG** [comment]
GZYZG+ 2227423992 + 空间变压器网络教程

顺带问一句：可以修改教程的标题吗，比如 "空间变压器网络教程“ ？😁

**Comment by jiangzhonglian** [comment]
> GZYZG+ 2227423992 + 空间变压器网络教程
> 
> 顺带问一句：可以修改教程的标题吗，比如 "空间变压器网络教程“ ？😁

可以的

**Comment by jiangzhonglian** [comment]
> 真想不出昵称+2445657635+torchvision

加油！别忘记更新哈！

**Comment by jiangzhonglian** [comment]
> ethan4335 + 309335077 + 分布式数据并行入门

加油！别忘记更新哈！

**Comment by jiangzhonglian** [comment]
> SAGIRI + 1900384123 + 什么是PyTorch？

加油！别忘记更新哈！

**Comment by jiangzhonglian** [comment]
> haiyang + 1130248527 + 文本部分剩余章节

加油！别忘记更新哈！

**Comment by baibizhe** [comment]
baibizhe+1446977280+CUDA 语义
顺带问一句：除了QQ能有别的交流方式嘛...例如微信 or 倍洽(一个简洁的/扁平化的交流工具，国内版的slack)

**Comment by hardmstar** [comment]
hardmstar+348518961+torch.utils.data

**Comment by zhangqizky** [comment]
您的邮件已收到，我会尽快回复，谢谢。张琦

**Comment by zanshuxun** [comment]
请问1.7的进度在哪
